crates/api/src/constraints/api.rs

```diff
@@ -225,22 +274,31 @@ where
         let body = req.into_body();
         let body_bytes = to_bytes(body, MAX_REQUEST_LENGTH).await?;
         
-        // Decode the incoming request body into a `SignedDelegation`.
-        let mut signed_revocation: SignedRevocation = match serde_json::from_slice(&body_bytes) {
-            Ok(revocation ) => revocation,
-            Err(_) => return Err(ConstraintsApiError::InvalidRevocation),
+        // Decode the incoming request body into a `SignedRevocation`.
+        let mut signed_revocation = match serde_json::from_slice::<SignedRevocation>(&body_bytes) {
+            Ok(revocation ) => match revocation.message.action {
+                1 => revocation,
+                other => {
+                    warn!(request_id = %request_id, action = other, "Invalid revocation action. expected 1");
```

Comment:
Can we bind `1` to a constant `REVOCATION_ACTION` with link to reference in the documentation?